### PR TITLE
Move DateTime rep mochitest to jest

### PR DESCRIPTION
**Pull request by arashm**
Don't have much experience on React, just tried to follow the tutorial in #415. Hope it's alright.

### CODE REVIEW

File: packages/devtools-reps/src/reps/stubs/date-time.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let stubs = new Map([]);
```
Comment (nchevobbe) [inline packages/devtools-reps/src/reps/stubs/date-time.js]: I think we can remove the array inside, my bad for having it in the guide and in the NaN stub

**Comment by nchevobbe** [comment]
Merged 🎉 Thanks again @arashm

**Comment by arashm** [comment]
Cool! So I pick another test.

**Comment by nchevobbe** [comment]
@arashm Maybe you could add a comment on the issue #415 to tell which test you're working on so other contributor don't take the same one :)
I'll add that in the guide too

**Comment by arashm** [comment]
Good idea. Will do as you said.

**Pull request merged**